src/components/MovieCard2.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+    <section :id="idName" :class="`${idName}`">
+        <div>
+            <slot name="image" :class="`${idName}-image`"></slot>
+        </div>
+        <div :class="`${idName}-header`">
+            <slot name="header"></slot>
+        </div>
+        
+        <div :class="`${idName}-genre`">
+            <slot name="genre"></slot>
+        </div>
+        <div :class="`${idName}-length`">
+            <slot name="length"></slot>
+        </div>
+        <div :class="`${idName}-screenings`">
+            <slot name="screenings"></slot>
+        </div>
+    </section>
+</template>
+
+<script>
+export default {
+    name: "MovieCard2",
+    props: ['idName'],
+}
+</script>
```

Comment:
Looks good for me, but there's one thing that I'm wondering, is every `MovieCard` item will have similar content: header, image, genre, length, screenings or this content can be different for each movie. If not then I think here it would be better to use props instead slot in this case(if all those blocks repeat in the other movie card), what do you think? :)

Reply:
I wanted to use it in two places: in one there will be header, title, genre and length and in the second one there will be additional screenings section and I was wondering if slot are good for that or are they too much?